model-optimizer/extensions/front/YOLO.py

```diff
@@ -51,12 +69,150 @@ def transform_graph(self, graph: Graph, replacement_descriptions):
                             'Refer to documentation about converting YOLO models for more information.'.format(
                     ', '.join(replacement_descriptions['entry_points']), input_node_name))
             last_node = Node(graph, input_node_name).in_node(0)
+            anchors = replacement_descriptions['anchors']
+
             op_params = dict(name=last_node.id + '/YoloRegion', axis=1, end_axis=-1, do_softmax=0)
             op_params.update(replacement_descriptions)
             if 'masks' in op_params:
                 op_params['mask'] = op_params['masks'][i]
+                anchors = anchors[op_params['mask'][0] * 2 : op_params['mask'][-1] * 2 + 2]
```

Comment:
Can you please print example to understand that it works?
```python
print(anchors)
print(op_params['mask'])
anchors = anchors[op_params['mask'][0] * 2 : op_params['mask'][-1] * 2 + 2]
print(anchors)
```

Reply:
```
[10, 13, 16, 30, 33, 23, 30, 61, 62, 45, 59, 119, 116, 90, 156, 198, 373, 326]
[6, 7, 8]
[116, 90, 156, 198, 373, 326]
[10, 13, 16, 30, 33, 23, 30, 61, 62, 45, 59, 119, 116, 90, 156, 198, 373, 326]
[3, 4, 5]
[30, 61, 62, 45, 59, 119]
[10, 13, 16, 30, 33, 23, 30, 61, 62, 45, 59, 119, 116, 90, 156, 198, 373, 326]
[0, 1, 2]
[10, 13, 16, 30, 33, 23]
```